docs/definitions/buyer_summary.csv

```diff
@@ -5,12 +5,12 @@ collection_id,integer,id from kingfisher collection table
 ocid,text,ocid from the data
 release_id,text,Release id from the data. Relevent for releases and not for compiled_releases or records
 data_id,integer,"id for the ""data"" table in kingfisher that holds the original JSON data."
-buyer,jsonb,
-buyer_parties_id,text,
-buyer_identifier,text,
-unique_identifier_attempt,text,
-buyer_additionalidentifiers_ids,jsonb,
-buyer_additionalidentifiers_count,integer,
-link_to_parties,integer,
-link_with_role,integer,
-party_index,bigint,
+buyer,jsonb,JSONB of buyer object
+buyer_parties_id,text,`id` from buyer object
+buyer_identifier,text,Concatenation from `scheme` and `id` from `identifier` object in the form `<scheme>-<id>`
+unique_identifier_attempt,text,"The `id` from buyer object if exists, otherwise the identifier described above if exists, otherwise the party name"
+buyer_additionalidentifiers_ids,jsonb,JSONB list of the concatenating scheme and id of all additionalidentifier objects
```

Comment:
concatenat*ing* scheme?

Reply:
To be consistent with elsewhere, and to clarify, I think this is clearest:

JSONB list of the concatenation of scheme and id